### PR TITLE
Compact description

### PR DESCRIPTION
**Pull request by anderseknert**
* Additionally, move "Contributing" section to bottom

Signed-off-by: Anders Eknert <anders@eknert.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,12 @@
...
-## Contributing
+Rego. From Styra, the founders of Open Policy Agent (OPA), and some of the most experienced members of the community,
+we hope to share some lessons learnt from authoring hundreds of thousands of lines of production policy over the years.
 
-This document is meant to reflect the style preferences and best practices as compiled by the OPA community. As such,
-we welcome contributions from any of its members. Since most of the topics in a guide like this are likely subject to
-discussion, please open an issue, and allow some time for people to comment, before opening a PR.
-
-If you'd like to add or remove items for your own company, team or project, forking this repo is highly encouraged!
+With new features, language constructs and other improvements continuously finding their way into OPA, we aim to keep
```
Comment (philipaconrad) [inline README.md]: Consider (Oxford comma): new features, language constructs, and other improvements

**Review by philipaconrad — COMMENTED** [review]
Overall, looks good! :smile: I'll let Torin comment more on the substance of it.

The only thing I noticed in proofreading was a spot where the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) style might make sense. If you prefer the British English style where serial commas are optional, feel free to disregard my comment, since it's fairly clear what was meant in this context.

File: README.md (hunk 1 above)
Comment (anderseknert) [reply README.md]: Fixed, thanks :)

File: README.md [hunk 2]
```diff
@@ -1,33 +1,12 @@
...
-of developers, finding acceptance (if not consensus) on a set of principles is often more important than the principles
-themselves. Use the rules provided here as you wish: adopt the guide in its entirety, pick what you like, or go your
-own way.
-
-Since best practices change over time, make sure to check back every now and then. We intend to keep this project
-versioned, with any changes published included in the `CHANGELOG.md` file. This should allow you to quickly scan for
-updates in recommendations since you last checked in.
-
-## Contributing
+Rego. From Styra, the founders of Open Policy Agent (OPA), and some of the most experienced members of the community,
```
Comment (tsandall) [inline README.md:4]: nit: drop "modern"

File: README.md [hunk 3]
```diff
@@ -1,33 +1,12 @@
...
+we hope to share some lessons learnt from authoring hundreds of thousands of lines of production policy over the years.
```
Comment (tsandall) [inline README.md:5]: Maybe instead...


```
we hope to share lessons learnt from authoring and reviewing hundreds of thousands of lines of Rego over the years.
```

File: README.md (hunk 2 above)
Comment (anderseknert) [reply README.md:4]: 👍

File: README.md (hunk 3 above)
Comment (anderseknert) [reply README.md:5]: Thanks, changed.

**Pull request merged**